examples/memoryManager.hpp

```diff
@@ -0,0 +1,38 @@
+#ifndef __MEMORYMANAGER_HPP__
+#define __MEMORYMANAGER_HPP__
+
+#include "RAJA/RAJA.hpp"
+#include "RAJA/util/defines.hpp"
+
+/*
+  Developer must manage memory
+*/
+namespace memoryManager{
```

Comment:
Can you comment this file fully, it would be a good place to explain how RAJA interacts with data.

Reply:
Thanks @davidbeckingsale, I think I got them all now. @trws @rhornung67, if there is anymore comments or suggestions please let me know.